Tests/SwiftGRPCTests/ClientTestExample.swift

```diff
@@ -92,6 +92,32 @@ extension ClientTestExample {
     XCTAssertEqual([Echo_EchoRequest(text: "foo")], fakeService.getRequests)
   }
   
+  func testClientAsynchronous() throws {
+    let fakeService = Echo_EchoServiceTestStub()
+    fakeService.getResponses.append(Echo_EchoResponse(text: "bar"))
+    
+    let client = ClientUnderTest(service: fakeService)
+    
+    let completionHandlerExpectation = expectation(description: "request completion handler called")
+
+    _ = try client.service.get(Echo_EchoRequest(text: "foo")){ (response:Echo_EchoResponse?, callResult:CallResult) in
+      
+      XCTAssertEqual("bar", response?.text)
+      completionHandlerExpectation.fulfill()
+    }
+    
+    waitForExpectations(timeout: 1) { (error) in
```

Comment:
Why not just `waitForExpectations(timeout: defaultTimeout)`, without the block?

Reply:
There is not a `defaultTimeout` should I do `ClientTestExample: BasicEchoTestCase` so I can inherit?

Reply:
Sorry, missed that. Just use `waitForExpectations(timeout: 1)`, without the closure, then.